ui/src/groups/GroupSidebar/GroupSidebar.tsx

```diff
@@ -95,9 +98,16 @@ function GroupHeader() {
       </GroupActions>
       <Link
         to=".."
-        className="h-6-w-6 absolute top-2.5 left-2 z-40 flex items-center justify-center rounded bg-white bg-transparent p-1 text-gray-400"
+        className={cn(
+          'h-6-w-6 absolute top-2.5 left-2 z-40 flex items-center justify-center rounded bg-white bg-transparent p-1 text-gray-400',
+          needsUpdate ? 'bg-yellow' : 'bg-transparent'
+        )}
       >
-        <CaretLeft16Icon className="h-4 w-4" />
+        {needsUpdate ? (
+          <AsteriskIcon className="h-4 w-4 text-black dark:text-white" />
```

Comment:
will this look right on dark with yellow + white?

Reply:
oh nvm ignore, I forget they're flipped

Reply:
`text-black`/`text-white` have inverted meaning when theme is dark. Counterintuitively, this forces it to always render as black, not sure if there's a cleaner way to do it